CMakeLists.txt

```diff
@@ -133,6 +133,31 @@ endif()
 find_package(Threads REQUIRED)
 find_package(ZLIB REQUIRED)
 
+# try to find an installed boost or install locally with conan
+set(boost_VERSION "1.71")
+# find_package(Boost ${boost_VERSION} QUIET)
+# if (NOT Boost_FOUND)
```

Comment:
this looks like you were testing and accidentally didnt put it back. ill give it a try